samples/kernel/metairq_dispatch/README.rst

```diff
@@ -63,3 +63,84 @@ preemption behavior, it does not run without modification on
 native_posix platforms.  In that emulation environment, threads will
 not be preempted except at specific instrumentation points (e.g. in
 k_busy_wait()) where they will voluntarily release the CPU.
+
+Building and Running
+********************
+
+This application can be built and executed on frdm_k64f as follows:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/kernel/metairq_dispatch
+   :board: frdm_k64f
+   :goals: run
```

Comment:
`run` is used for qemu, but you need to use `flash` for real hardware
```suggestion
   :goals: build flash
```

Reply:
Fixed.